src/v2/view-builder/views/password/ResetAuthenticatorPasswordView.js

```diff
@@ -0,0 +1,22 @@
+import { loc } from 'okta';
+import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
+
+const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
+
+  className: 'password-authenticator',
+
+  title () {
+    return loc('password.reset.title.generic', 'login');
+  },
+
+  save () {
+    return loc('password.reset', 'login');
+  },
+
+  getPasswordPolicy () { 
```

Comment:
just thought and not necessarily make changes now.

Feels like the default implementation of `getPasswordPolicy` in `EnrollAuthenticatorPassword` can be

```
return this.options.appState.get('currentAuthenticator').settings;
```

as it covers most of case. one override reads `currentFactor` is probably not correct for M1 and will be addressed soon.

Reply:
Absolutely! I was going to address it in my next ticket (https://oktainc.atlassian.net/browse/OKTA-308791) and wanted to talk to you about it before I do, I will use `currentAuthenticator` in `EnrollAuthenticatorPassword`.

Another question - I am currently using `recoveryFactor` for `password expired scenario`, does it make sense to change it to `recoveryAuthenticator` or do I need to use both since `recoveryAuthenticator` is only present when `AUTHENTICATOR_PLATFORM` is enabled. Ref: https://okta.slack.com/archives/CCA77QVL2/p1593714897203000

Reply:
Although, even if it is the same, I'd rather override it here and have the same code. Because if logic to get password policy in `EnrollAuthenticatorPassword` changes, it will break `ResetAuthenticatorPassword` if not overridden, any thoughts?

Reply:
> I am currently using recoveryFactor for password expired scenario

It's probably going to be `recoveryAuthenticator`.

> Although, even if it is the same, I'd rather override it here and have the same code

I cannot tell if it's possible that logic in `EnrollAuthenticatorPassword` changed but not others. (a.k.a `EnrollAuthenticatorPassword` doesn't read `recoveryAuthenticator` but other child class.).
It feels to me unlikely happen. But I'm fine to keep the override in sub class now.

Reply:
oh I see it now. it reads `currentAuthenticator` during EnrollPassword but `recoveryAuthenticator` at other cases.